apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -814,6 +812,7 @@ test.describe("Text area min and max characters text", () => {
       // Expect the native <input> element to show an error message
 
       let validationMessage = await minInput?.evaluate((input: any) => input?.validationMessage as string);
+      // FIXME: This error message will be localized depending on the browser locale.
```

Comment:
This works on CI but locally my browser is in Spanish. Causing this to fail even if I have my locale set to "en-US". I don't know how to tell PW to install the browser in a specific locale.

![image](https://github.com/user-attachments/assets/0fa0806e-9409-4c62-bd49-96baa209eb58)

Reply:
Hey, are you trying to change the locale inside the use block of playwright.config.ts?

Reply:
> Hey, are you trying to change the locale inside the use block of playwright.config.ts?

It is already set to English. But in my machine the browser itself sometimes installs as Spanish. See the [linked issue for more details](https://github.com/microsoft/playwright/issues/17326#issuecomment-2332690718).